advanced/Scripts/webpage.sh

```diff
@@ -401,22 +401,38 @@ SetWebUILayout() {
     change_setting "WEBUIBOXEDLAYOUT" "${args[2]}"
 }
 
+CheckUrl(){
+    local url validUrl regex
+    #regex from https://www.regextester.com/94502
+    regex="^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$"
```

Comment:
We also support `file://` as protocol, maybe even `ftp://` (?). This regex only permits `http(s)://` or nothing.

Reply:
Possibly `///` shorthand as well.